### PR TITLE
Feature/2702 watch ext

### PR DESCRIPTION
**Pull request by amenk**

### CODE REVIEW

File: src/org/traccar/model/Command.java [hunk 1]
```diff
@@ -41,6 +41,7 @@
     public static final String TYPE_OUTPUT_CONTROL = "outputControl";
     public static final String TYPE_VOICE_MONITORING = "voiceMonitoring";
     public static final String TYPE_SET_AGPS = "setAgps";
+    public static final String TYPE_SET_HEARTS = "setHearts";
```
Comment (tananaev) [inline src/org/traccar/model/Command.java]: What does this command do? It doesn't sound generic enough.

Comment (tananaev) [inline src/org/traccar/model/Command.java]: I guess something like that. Also, please use `KEY_DATA` for the value.

**Comment by amenk** [comment]
All done I believe. Thank you for your review!

File: src/org/traccar/protocol/WatchProtocol.java [hunk 2]
```diff
@@ -40,7 +41,10 @@ public WatchProtocol() {
                 Command.TYPE_SILENCE_TIME,
                 Command.TYPE_ALARM_CLOCK,
                 Command.TYPE_SET_PHONEBOOK,
-                Command.TYPE_VOICE_MESSAGE);
+                Command.TYPE_VOICE_MESSAGE,
+                Command.TYPE_SET_TIMEZONE,
+                Command.TYPE_SET_INDICATOR
+            );
```
Comment (tananaev) [inline src/org/traccar/protocol/WatchProtocol.java]: Please move this bracket to the previous line. Everything else looks good.

**Comment by tananaev** [comment]
It looks like you changed it to minutes, not seconds.

**Comment by amenk** [comment]
you are right :) - fixed

**Comment by tananaev** [comment]
Merged, thanks for pull request.

**Pull request merged**